### PR TITLE
feat: add docker-compose file for enable metrics pipeline

### PR DESCRIPTION
**Pull request by kotharironak**
As part of this ticket - https://github.com/hypertrace/hypertrace/issues/327 - on the context of providing application metrics support, this PR,

- adds the new docker-compose file for metrics pipeline
- corresponding hypertrace-ingester PR - https://github.com/hypertrace/hypertrace-ingester/pull/292

### CODE REVIEW

**Comment by kotharironak** [comment]
will merge this after - https://github.com/hypertrace/hypertrace-ingester/pull/292

**Review by rish691 — COMMENTED** [review]
nit: can we add a note in the readme?

**Comment by jcchavezs** [comment]
A couple of comments here:

1. Do we really want to add a traceable image in this project?
traceableai-docker.jfrog.io/hypertrace/hypertrace-ingester:test also it
looks like the tag isnt too stable. Maybe we want to publish the image in
hypertrace org?
2. Please don't use `container_name` it is truly a pain in local because it
will collide with other local setups using such container name.
3. Shall we add health check rules for Prometheus container?

**Comment by kotharironak** [comment]
> nit: can we add a note in the readme?

Done

**Comment by kotharironak** [comment]
> A couple of comments here: 1. Do we really want to add a traceable image in this project? traceableai-docker.jfrog.io/hypertrace/hypertrace-ingester:test also it looks like the tag isnt too stable. Maybe we want to publish the image in hypertrace org? 2. Please don't use `container_name` it is truly a pain in local because it will collide with other local setups using such container name. 3. Shall we add health check rules for Prometheus container?

> 1. Do we really want to add a traceable image to this project?

This was accidental. was testing with locally built images. Removed.

> 2. Please don't use `container_name` it is truly a pain in local because it will collide with other local setups using such container name. 

Done

File: docker/README.md
Comment (rish691) [inline docker/README.md:48]: user also have to override the query service config to read from prometheus?

Comment (kotharironak) [reply docker/README.md:48]: Can we extend that once QS part all merged?

Comment (rish691) [reply docker/README.md:48]: yeah sure

**Review by rish691 — APPROVED** [review]

**Pull request merged**
